astro-client/astro.go

```diff
@@ -33,47 +32,48 @@ type Client interface {
 	CreateUserInvite(input CreateUserInviteInput) (UserInvite, error)
 }
 
-func (c *HTTPClient) ListUserRoleBindings() ([]RoleBinding, error) {
+func (c *HTTPClient) SelfUser() (*Self, error) {
```

Comment:
```suggestion
func (c *HTTPClient) GetUserInfo() (*Self, error) {
```
nitpick: won't `GetUserInfo` better describe this function then `SelfUser`

Reply:
What do you think?

Reply:
Makes sense. @sunkickr Feel free to update the function name

Reply:
okay